sound/soc/sof/topology.c

```diff
@@ -932,6 +926,122 @@ static int sof_parse_uuid_tokens(struct snd_soc_component *scomp,
 	return found;
 }
 
+/**
+ * sof_copy_tuples - Parse tokens and copy them to the @tuples array
+ * @sdev: pointer to struct snd_sof_dev
+ * @array: source pointer to consecutive vendor arrays in topology
+ * @array_size: size of @array
+ * @token_id: Token ID associated with a token array
+ * @sets: number of times the same token needs to be parsed i.e. the function looks for
+ *	  "sets" instances of each token in the token array associated with the token ID.
```

Comment:
quite confusing. 'set' is not usually used for counters or instance numbers.

how about:
token_instance_num: number of times the same @token_id needs to be parsed i.e. the function looks for
@token_instance_num of each token in the token array associated with the @token_id

is the token array "@array" ? what is the "token array"

Reply:
done. and not token array is not @array. Each token_id is associated with an array or pre-defined tokens. The array is the vendor array in topology.

Reply:
is this updated? I still see the same thing?